cmd/main.go

```diff
@@ -37,25 +37,23 @@ import (
 )
 
 var (
-	bqDataset        = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
-	btProject        = flag.String("bt_project", "", "GCP project containing the BigTable instance.")
-	baseBtInstance   = flag.String("base_bt_instance", "", "Base cache BigTable instance.")
-	baseTableName    = flag.String("base_table", "", "Base cache Bigtable table.")
-	branchBtInstance = flag.String("branch_bt_instance", "", "Branch cache BigTable instance.")
-	projectID        = flag.String("project_id", "", "The cloud project to run the mixer instance.")
-	port             = flag.Int("port", 12345, "Port on which to run the server.")
-	useALTS          = flag.Bool("use_alts", false, "Whether to use ALTS server authentication")
-	bigqueryOnly     = flag.Bool("bigquery_only", false, "The service only serves sparql query")
-	svobsMode        = flag.Bool("svobs_mode", false, "Use new storage that is built on top of StatVar instead of StatPop")
-	schemaPath       = flag.String("schema_path", "/translator/mapping", "The director that contains the schema mapping files")
+	mixerProject  = flag.String("mixer_project", "", "The cloud project to run the mixer instance.")
+	storeProject  = flag.String("store_project", "", "GCP project stores Bigtable and BigQuery.")
+	bqDataset     = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
+	baseTableName = flag.String("base_table", "", "Base cache Bigtable table.")
+	port          = flag.Int("port", 12345, "Port on which to run the server.")
+	useALTS       = flag.Bool("use_alts", false, "Whether to use ALTS server authentication")
+	bigqueryOnly  = flag.Bool("bigquery_only", false, "The service only serves sparql query")
+	svobsMode     = flag.Bool("svobs_mode", false, "Use new storage that is built on top of StatVar instead of StatPop")
+	schemaPath    = flag.String("schema_path", "/translator/mapping", "The director that contains the schema mapping files")
```

Comment:
Would we not have different mapping files based on svobsMode ?

Reply:
This is a mapped directory in the pod. It's the same for all deployment: https://github.com/datacommonsorg/mixer/blob/master/deploy/base/deployment.yaml#L64-L66, which is specified by the config map https://github.com/datacommonsorg/mixer/blob/master/deploy/base/deployment.yaml#L46-L47. For example, in the current deployment: https://github.com/datacommonsorg/mixer/blob/master/deploy/mapping/kustomization.yaml#L18